Website/src/rendering/src/components/Sponsors/SelectedSponsorsGrid.tsx

```diff
@@ -0,0 +1,34 @@
+import Link from 'next/link';
+import { Image, withDatasourceCheck } from '@sitecore-jss/sitecore-jss-nextjs';
+import { ComponentProps } from 'lib/component-props';
+import { Sponsor } from 'src/types/sponsor';
+
+export type SponsorsProps = ComponentProps & {
```

Comment:
I realize the previous `SponsorsGrid` component had a props type with an incoherent name. We try to name the prop type with `[ComponentName]Props`. So this one should be named `SelectedSponsorsGridProps`.